src/modules/dashboard/components/Extensions/index.ts

```diff
@@ -1,2 +1,10 @@
+import { Extension } from '@colony/colony-js';
+
 export { default } from './Extensions';
 export { default as ExtensionDetails } from './ExtensionDetails';
+
+// List of extensions allowed for use within the Dapp
+export const dappExtensions = [
+  Extension.OneTxPayment,
+  Extension.VotingReputation,
+];
```

Comment:
So I think we can simplify this even further.

1. Remove this
2. Remove `Unknown` from the `extensions` map inside `data/staticData/extensionData.tsx` _(it was set up as an fallback, but we never ended up using it)_
3. Refactor the `extensions` map to use `Extension.<ExtensionName>` enums as keys _(instead of them being manually typed)_
4. Use that array as the source for what extensions to surface in the dapp

It actually makes sense since if that mapping doesn't have an entry for the extension, it won't have any data to display when trying to render the extension's card on the extension's page.

So it's only fitting that we use this object to also control the "supported" extensions inside the dapp 

_(maybe leave a comment about this, that we're using this object to control the extensions display)_